lib/whitehall_importer/integrity_checker/change_history_check.rb

```diff
@@ -0,0 +1,47 @@
+module WhitehallImporter
+  class IntegrityChecker::ChangeHistoryCheck
+    attr_reader :live_edition, :proposed_change_history, :publishing_api_change_history
+
+    def initialize(live_edition, proposed_change_history, publishing_api_change_history)
+      @live_edition = live_edition
+      @proposed_change_history = proposed_change_history
+      @publishing_api_change_history = publishing_api_change_history
+    end
+
+    def match?
+      return false unless history_length_matches?
+
+      if !live_edition
```

Comment:
Tiny one, but I think it's an easier read if it's `if live_edition...else`, which will mean a switch around of the methods that are called.